test/functional/apps/discover/_discover.ts

```diff
@@ -193,8 +192,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('should show "no results"', async () => {
-        const isVisible = await PageObjects.discover.hasNoResults();
-        expect(isVisible).to.be(true);
+        await retry.waitFor('No results screen is displayed', async function () {
```

Comment:
:art: description should starts with lowercase letter here

Reply:
I do agree! thx, changed that!